testdata/p4_16_samples/ebpf/key-issue-1020_ebpf.p4

```diff
@@ -23,7 +23,7 @@ parser prs(packet_in p, out Headers_t headers) {
     }
 }
 
-control pipe(inout Headers_t headers, out bool pass) {
+control key-issue-1020_ebpf(inout Headers_t headers, out bool pass) {
```

Comment:
This is not even legal P4. This should not pass compilation.
This proves that your testing is actually broken - the tests should not pass with this control name.
Please make sure that when you submit something make check works.
Because you moved the ebpf files to a different directory some of the old tests are no longer running.